### PR TITLE
Name Request UI updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6333

*Note:* 
*DO NOT MERGE:  WAITING on Entity Selector Tool to be deployed, so we can link to it. 

*Description of changes:*

* General visual and verbiage updates.
* Updated Tabs to appear more 'tab-like'.
* Added a `link-row` to the bottom of New request search.
* Added helper info to Existing request tab.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/existing-request/existing-request-search.vue [hunk 1]
```diff
@@ -1,68 +1,107 @@
...
-                      label="NR Number"
-                      validate-on-blur />
+      <!-- Help Info -->
+      <v-col cols="3" class="existing-request-info copy-small">
+        <div class="ma-8">
+          <ul class="mt-3">
+            <li>
+              Check review time and approval status
+            </li>
+            <li class="test-arrow">
```
Comment (severinbeauvais) [inline client/src/components/existing-request/existing-request-search.vue:12]: Is this class used?
(Also, how did you implement the arrow?)

File: client/src/components/tabs.vue [hunk 2]
```diff
@@ -61,17 +67,31 @@ export default class Tabs extends Vue {
...
   border-top-right-radius: 4px;
-  min-height: 58px;
-  max-height: 58px;
+}
+
+::v-deep .theme--light.v-tabs > .v-tabs-bar{
+  background-color: transparent;
+}
+
+::v-deep .theme--light.v-tabs > .v-tabs-bar .v-tab:not(.v-tab--active) > .v-icon {
```
Comment (severinbeauvais) [inline client/src/components/tabs.vue:93]: Whoa!
:)

**Review by severinbeauvais — APPROVED** [review]

File: client/src/components/existing-request/existing-request-search.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/existing-request/existing-request-search.vue:12]: sorry, was fiddling here. This needs to be removed. 

I added the arrow to the parent column using `:after`

File: client/src/components/tabs.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/components/tabs.vue:93]: I know 😩 
Manipulating Vuetify to meet the requirements can sometimes get a little 😬

Comment (severinbeauvais) [reply client/src/components/tabs.vue:93]: I noticed in the Vue docs that some global (CSS?) variables can be set to override some default styles. Not sure if that applies here... Don't worry about it; we can look into it later.

File: client/src/components/existing-request/existing-request-search.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/components/existing-request/existing-request-search.vue:12]: Cool.
How about adding a comment to the CSS block to say what it's doing?

Comment (cameron-freshworks) [reply client/src/components/existing-request/existing-request-search.vue:12]: Absolutely, will do!

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Pull request merged**